middlewares/validators/user.js

```diff
@@ -30,6 +31,7 @@ const createUser = async (req, res, next) => {
 
 const updateUser = async (req, res, next) => {
   const schema = joi.object().keys({
+    user_name: joi.string().optional(),
```

Comment:
Shouldn't we be making `user_name` and other fields like `yoe`, `company` required for the `updateUser` function? Otherwise someone may set `incompleteUserDetails` to false through postman by calling this API and get away without filling the form which sujit is working on.

Reply:
- One can't update `incompleteUserDetails` field because we don't have validation for it. Validator won't allow that field in data.
- We should not set other fields as `required`, because if I only want to change my YOE I won't send other data. User does not update the whole information all the time, only required fields are updated which we can't predict for each user.